.tekton/main-build.yaml

```diff
@@ -42,7 +42,7 @@ spec:
   - name: prefetch-input
     value: |
       [
-        { "type": "npm", "path": ".konflux/bootstrap-yarn" },
+        { "type": "npm", "path": "ui/apps/platform" },
         { "type": "gomod", "path": "." }
```

Comment:
Following [the Slack thread](https://redhat-internal.slack.com/archives/C7WH69HCY/p1722515757546739?thread_ts=1722451493.032579&cid=C7WH69HCY), if the fix turns to be a feature (<https://issues.redhat.com/browse/KONFLUX-3917>) that will not emerge short-term, I suggest we should disable `npm` prefetch for the time being to not break the builds.
```ayml
  - name: prefetch-input
    # TODO(ROX-25572): re-add { "type": "npm", "path": "ui/apps/platform" },
    value: |
      [
```

By now we know that we can prefecth, we just can't enable it for as long as source container build is failing.

Reply:
100% agree. If there isn't an immediate resolution on the Quay side we can disable this for now and create a follow up ticket and draft PR to re-enable later.

Reply:
https://github.com/stackrox/stackrox/pull/12316